### PR TITLE
Include command line executed in LogCmd error

### PR DESCRIPTION
**Pull request by vcaputo**

### CODE REVIEW

File: internal/log/log.go [hunk 1]
```diff
@@ -115,21 +115,33 @@ func (l *Logger) PopPrefix() {
 	l.prefixStack = l.prefixStack[:len(l.prefixStack)-1]
 }
 
+// quotedCmd returns a concatenated, quoted form of cmd's cmdline
+func quotedCmd(cmd *exec.Cmd) string {
+	if len(cmd.Args) == 0 {
+		return fmt.Sprintf("%q", cmd.Path)
+	}
+
+	var q []string
```
Comment (crawford) [inline internal/log/log.go]: Seed with cmd.Path?

Comment (vcaputo) [inline internal/log/log.go]: thought about it, but this is getting bogged down in the minutia of os/exec.  We're the only users of this, and we know we're never supplying an os.Path != os.Args[0], so why bother?

Comment (crawford) [inline internal/log/log.go]: Now I understand. This go interface is absolutely brain-dead.

**Comment by crawford** [comment]
LGTM

**Pull request merged**